<launch>
	<!-- param -->
	<param name="use_sim_time" value="true"/>

	<!-- rosbag -->
	<node pkg="rosbag" type="play" name="player" args="--clock /home/amsl/ozaki/bagfiles/sq2/outdoor_dkan/2018-12-11-13-37-56.bag -r 1"/>
	
	<!-- rviz -->
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find attitude_estimation_walls)/rviz_config/attitude_estimation_walls.rviz"/>

	<!-- sensor -->
	<!-- <node pkg="sq_lidar" name="sq_lidar" type="sq_lidar"> -->
	<!-- 	<param name="fixed_frame" value="/base_link"/> -->
	<!-- 	<remap from="~/scan" to="/sq_lidar/scan"/> -->
	<!-- 	<remap from="~/cloud" to="/cloud"/> -->
	<!-- </node> -->

	<!-- MAIN -->
	<node pkg="attitude_estimation_walls" name="attitude_estimation_walls_ekf" type="attitude_estimation_walls_ekf" output="screen">
	</node>

</launch>
